### ISSUE TITLE
In Sound Mind (1119980)

### ISSUE THREAD

**Issue by ThisNekoGuy** [ValveSoftware/Proton#7283]
# Compatibility Report
- Name of the game with compatibility issues: In Sound Mind
- Steam AppID of the game: 1119980

## System Information
- GPU: RX 7800 XT
- Video driver version: Mesa 23.3.0-rc4 (RADV)
- Kernel version: [6.5.9-tkg-pds-gentoo-gcc-zen2](https://github.com/frogging-family/linux-tkg)
- Link to full system information: [Gist](https://gist.github.com/ThisNekoGuy/9b004c8b54c8e9ea1f7646cef51447fc)
- Proton version: 8.0-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

Log: [steam-1119980.zip](https://github.com/ValveSoftware/Proton/files/13455287/steam-1119980.zip)
[In Sound Mind - LocalLow AppData Folder (Sava Data).zip](https://github.com/ValveSoftware/Proton/files/13455301/In.Sound.Mind.-.LocalLow.AppData.zip)

## Symptoms 
When using the second obtained tape in the game (required for progression) in the player's office, completing the memory scene, and exiting the associated area through a door the game crashes with a consistent reproduction rate.

## Reproduction

1) Play the game until you get a second audio cassette tape (or use my provided save data) - (This is relatively early in the game)
2) Load the tape into the player in the office
3) Press the play button
4) Leave the office
5) Do the memory scene (you'll find more tape players that actually play audio), navigate the area
6) Leave the scene though a door
7) Triggers the crash during the loading screen

**Comment by totesido** [comment]
Hi, I'm part of the original ISM dev team. I'm not sure how to help here but in case this ever gets looked at, to give more context: the second tape in the game (after the "memory scene") is basically when you first enter a big level in the game, after the introduction level in the building. It's roughly 10 minutes in. It's the "supermarket" level where you meet The Watcher, a ghost with a tragic story. 

If I had to guess, this is the point it's crashing because it's loading shaders unique to that chapter (and of The Watcher). We used Unity 2020.3 and HDRP 10 for the production of this game and it's a pretty early version of HDRP (despite not being experimental for 2 years at this point). On PC Windows, it's been a relatively stable, but pretty much every single non-Windows port had major stability issues with custom HDRP shaders initially. We had to make all sorts of modifications around our custom shaders in ported versions for a stable release. We're not actively developing the original ISM at this point and I'm not a tech art expert (I was the producer/ui programmer), but feel free to ping me if you need more information. Thanks for playing ISM!

Oh also worth mentioning, if it helps eliminating the source of the issue, last I checked (few months ago), Steam Deck runs ISM start to finish.

**Comment by ThisNekoGuy** [comment]
> Thanks for playing ISM!
Oh also worth mentioning, if it helps eliminating the source of the issue, last I checked (few months ago), Steam Deck runs ISM start to finish.

Thanks for letting me know; I'll continue it over there if it doesn't get fixed on desktop Linux soon :+1: 
I appreciate you taking the time to say something because you didn't even have to

**Comment by 825i** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/7283#issuecomment-1830281541

Hey @totesido nice to meet you. I am having a pretty serious issue with the game crashing like mad when I am back in my office talking to Tonia right after having just got the "bull" vinyl, and right before I start Lucas' tape. Basically the screen keeps flashing and blacking out in the office so I can't play the tape, the game is pretty much resizing the screen all weird. Like the resolution just drops or changes or bugs out. I pretty much start the game and it just bugs out within about 10 seconds. There's not enough time to load the tape and walk through the door. Even worse, now it seems the game won't even load. After crashing 3 times in a row I very briefly saw a little popup when running from Steam which looked like a file integrity checker or something. It looked like it was ISM's own one and not Steam's. I also tried verifying file integrity from Steam but it finds no issues at all. I think my shader cache is probably corrupt or I don't know.

I assume the root cause is something to do perhaps with the mirror? Using/holding the mirror in the office at this point in the game just seems to cause some weird stuff to happen. Generally the mirror already tanks FPS if you hold it to display things. Likely due to raytracing? or light bouncing or whatever. I can go from like 144fps to literally 70fps holding the mirror lol. I'm using an RTX 4090 with the 552.22 drivers. I've not had problems with Nivdia drivers for many years, but it may also be their fault too as I just updated them 2 days ago and this has happened only once before I think in ISM at a different part of the game but I was able to continue after a brief black screen flicker. Unfortunately now though it looks like the game won't load at all even after a reboot. So I suppose I have to do a re-install.

I doubt you'll ever release a fix for this, though who knows. I've seen some games come back and release massive fixes after literally 5-7 years. Look at [Darkwood](https://store.steampowered.com/app/274520/Darkwood/) for instance, another amazing horror game. They just released a gigantic update after so many years that fixed dozens of issues people had reported. So if you ever work on this game again or your team ever does, I hope you can remember this is an issue.

Oh and I hope you guys avoid Unity in the future, it's more trouble than its worth. Plus the new licensing changes are totally anti-consumer and anti-developer. ISM 2 or whatever would look far better in UE5.

**Comment by kisak-valve** [comment]
Hello @825i, you've come across an issue tracker for a project called Proton. Proton is a compatibility layer to run Windows based games on Linux. Using the NVIDIA 552.22 driver release tells us that you're using Windows instead of Linux. Running the game on Windows itself is out of scope for this issue tracker.

Please use the Steam community forums or whatever appropriate feedback medium requested by the game devs to communicate with the game devs.
